osmosis-1/osmosis.zone_assets.json

```diff
@@ -4850,6 +4850,19 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/uwu.sol.pica.png"
         }
       }
+    },
+    {
+      "chain_name": "haqq",
+      "base_denom": "erc20/0x4FEBDDe47Ab9a76200e57eFcC80b212a07b3e6cE",
+      "path": "transfer/channel-1575/erc20/0x4FEBDDe47Ab9a76200e57eFcC80b212a07b3e6cE",
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-08-15T12:00:00Z",
+      "peg_mechanism": "collateralized",
+      "categories": [
+        "stablecoin",
```

Comment:
@sunnya97 are we considering gold stables "stablecoins"? or is that category reserved for government currencies?